meetup/models.py

```diff
@@ -16,23 +16,32 @@ class Venue(models.Model):
 
 
 class ProgramCategory(models.Model):
-    """This model is for the category of program"""
+    """This model is the category of program"""
     name = models.CharField(max_length=100)
     slug = models.SlugField(max_length=100, unique=True)
 
 
 class Speaker(models.Model):
-    """This model is for that user related by speaker at program"""
+    """This model is that user related by speaker at program"""
     user = models.ForeignKey(User)
 
 
+class MeetUp(models.Model):
+    """This model is to manage each meet-up"""
+    title = models.CharField(max_length=100)
+    venue = models.ForeignKey(Venue)
+    when = models.DateField()
+    start_time = models.TimeField()
+    end_time = models.TimeField()
+
+
 class Program(models.Model):
-    """This model is for that lecture or event by speaker(s)"""
+    """This model is lecture or event by speaker(s)"""
     # There should be difficulty and language information in Program?
     title = models.CharField(max_length=255)
     brief = models.TextField(null=True, verbose_name='simple information')
     description = models.TextField(null=True)
-    speakers = models.ManyToManyField(Speaker, null=True)
+    speakers = models.ManyToManyField(Speaker)
     category = models.ForeignKey(ProgramCategory, null=True, verbose_name='The category which this program is belonged')
     slide_url = models.URLField(null=True, verbose_name='A slide url')
```

Comment:
늦은 질문 같긴 하지만, slide_url과 pdf_url을 구분하는 이유가 있나요?

Reply:
First, the `slide_url` is the url which is uploaded on presentation file services like speaker deck or slideshare by speaker.

And the `pdf_url` is a url which downloadable raw pdf file. Because there are some kind of people who want to watch the pdf file by their devices(maybe offline, during move).

Reply:
이건 정책의 문제같은데, slide_url로 충분해 보입니다. 우리가 raw pdf file을 제공하지 않을 예정입니다.

Reply:
I see, but we need to talk about this to other members on slack.